### ISSUE TITLE
duration as float?

### ISSUE THREAD

**Issue by reedwade** [mitchellh/go-server-timing#4]
Hi, looking at the W3C doc I'm hoping that duration can be a float?

I've got a mod for that at-- https://github.com/reedwade/go-server-timing/tree/float-duration

which causes the example to emit a header that looks like:

```
Server-Timing: service-4;dur=66.236352,service-2;dur=31.155601,service-1;dur=67.140114,service-3;dur=41.247412,service-0;dur=70.299034
```

This seems a lot more useful for a lot of server side things that happen quite fast (ideally).

**Comment by kasperlewau** [comment]
@reedwade 

I cherrypicked your changes and they're _beautiful!_

Now I get to see microseconds in the Chrome devtools as opposed to a fixed 0 value for anything less than 1ms. 

![image](https://user-images.githubusercontent.com/2687184/36254786-a782af60-124c-11e8-96be-fdcf9acdbcee.png)

**Comment by mitchellh** [comment]
This makes a ton of sense, please make your PR. :) 

It might make sense to limit to the # of decimal places to something like 2 or 3 though (chrome for example only shows 2).

**Comment by reedwade** [comment]
I'm inclined to keep the full precision as I'm also logging the header line on the server side for later review.

**Comment by mitchellh** [comment]
Fixed via #7
